apscheduler/executors/pool.py

```diff
@@ -44,10 +44,12 @@ class ThreadPoolExecutor(BasePoolExecutor):
     Plugin alias: ``threadpool``
 
     :param max_workers: the maximum number of spawned threads.
+    :param kwargs: dict of keyword arguments to pass to the underlying
+        ThreadPoolExecutor constructor
     """
 
-    def __init__(self, max_workers=10):
-        pool = concurrent.futures.ThreadPoolExecutor(int(max_workers))
+    def __init__(self, max_workers=10, **kwargs):
```

Comment:
I think I would prefer an actual dict parameter to be passed so that the job store can accept more parameters for itself too if deemed necessary.